README.md

```diff
@@ -107,6 +108,9 @@ However, there are some important limitations to note:
 2. Since all code is run on a remote driver through Livy, all structured data must be serialized to JSON and parsed by the Sparkmagic library so that it can be manipulated and visualized on the client side.
 In practice this means that you must use Python for client-side data manipulation in `%%local` mode.
 
+## Kerberos support
+Sparkmagic uses [requests-kerberos](https://github.com/requests/requests-kerberos) package for kerberos support. The sparkmagic expects kerberos ticket to be avaibale in the system. It picks up the kerbros ticket from cache file. User needs to run kinit to create the kerberos ticket. The sparkmagic doesn't support passing kerberos principal/token.
```

Comment:
Can we please change this paragraph to:

## Authentication Methods

Sparkmagic supports:

* No auth
* Basic authentication
* Kerberos

Kerberos support is implemented via the [requests-kerberos](https://github.com/requests/requests-kerberos) package. Sparkmagic expects a kerberos ticket to be available in the system. Requests-kerberos will pick up the kerberos ticket from a cache file. For the ticket to be available, the user needs to have run [kinit](https://web.mit.edu/kerberos/krb5-1.12/doc/user/user_commands/kinit.html) to create the kerberos ticket.

Currently, sparkmagic does not support passing a kerberos principal/token, but we welcome pull requests.